src/CMakeLists.txt

```diff
@@ -1009,9 +1030,9 @@ install(TARGETS osdmaptool DESTINATION bin)
 
 set(ceph_psim_srcs
   tools/psim.cc)
-add_executable(ceph-psim ${ceph_psim_srcs})
-target_link_libraries(ceph-psim global)
-install(TARGETS ceph-psim DESTINATION bin)
+add_executable(ceph_psim ${ceph_psim_srcs})
```

Comment:
@tchaikov (NIT) I was under the impression that executables had '-'s where spaces should go and libraries and other variables should have _'s where spaces should go. Does this make sense for consistency sake and if so can we change the packaging for ceph_psim to ceph-psim?